pkg/library/libraryapplyconfiguration/equivalence_test.go

```diff
@@ -0,0 +1,95 @@
+package libraryapplyconfiguration
+
+import (
+	"embed"
+	"github.com/google/go-cmp/cmp"
+	"io/fs"
+	"strings"
+	"testing"
+)
+
+//go:embed testdata
+var packageTestData embed.FS
+
+func must[A any](in A, err error) A {
+	if err != nil {
+		panic(err)
+	}
+	return in
+}
+
+func TestEquivalentApplyConfigurationResultIgnoringEvents(t *testing.T) {
+	type args struct {
+		lhsFS fs.FS
+		rhsFS fs.FS
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+		{
+			name: "honor fieldmanager in diff",
+			args: args{
+				lhsFS: must(fs.Sub(packageTestData, "testdata/need-useful-diff/lhs")),
+				rhsFS: must(fs.Sub(packageTestData, "testdata/need-useful-diff/rhs")),
+			},
+			want: []string{
+				`Management: rhs is missing equivalent request for fieldManager=openshift-authentication-Metadata controllerInstanceName=TODO-metadataController: ApplyStatus-Authentication.v1.operator.openshift.io/cluster[]`,
+				`Management: mutation: ApplyStatus-Authentication.v1.operator.openshift.io/cluster[], fieldManager=openshift-authentication-PayloadConfig, controllerInstanceName=TODO-payloadConfigController, rhs[1]: body diff:   []uint8(
```

Comment:
ok, this represents a request that matched on both sides, but the content was different